coordinator/tests/services/translation/demographics.spec.ts

```diff
@@ -112,35 +78,35 @@ describe("convertAddress should return correct addresses", () => {
     expect(() => demographics.convertAddress(fhirAddress)).toThrow(TypeError)
   })
 
-  test("address type as postal and use as anything else should return use as core.AddressUse.POSTAL", () => {
-    const fhirAddress = {type: "postal", use:"home", line: [""]}
+  const cases = [
+    ["home", {use: core.AddressUse.HOME}],
+    ["work", {use: core.AddressUse.WORK}],
+    ["temp", {use: core.AddressUse.TEMPORARY}]]
+
+  test.each(cases)("address type as postal and use as %p should return use as core.AddressUse.POSTAL", (argument: string) => {
+    const fhirAddress = {type: "postal", use:argument, line: [""]}
     const result = demographics.convertAddress(fhirAddress)
     expect(result._attributes).toEqual({use: core.AddressUse.POSTAL})
     expect(result.streetAddressLine).toHaveLength(1)
   })
 
-  test("address type not postal and use as allowed value should return correct value", () => {
-    const fhirAddressHome = {type: "example", use:"home", line: [""]}
-    const fhirAddressWork = {type: "example", use:"work", line: [""]}
-    const fhirAddressTemp = {type: "example", use:"temp", line: [""]}
-
-    const resultHome = demographics.convertAddress(fhirAddressHome)
-    const resultWork = demographics.convertAddress(fhirAddressWork)
-    const resultTemp = demographics.convertAddress(fhirAddressTemp)
-
-    expect(resultHome._attributes).toEqual({use: core.AddressUse.HOME})
-    expect(resultWork._attributes).toEqual({use: core.AddressUse.WORK})
-    expect(resultTemp._attributes).toEqual({use: core.AddressUse.TEMPORARY})
-  })
+  test.each(cases)(`address type not postal and use as %p should return correct value`,
+    (argument: string, expected) => {
+      const resultHome = demographics.convertAddress({type: "example", use:argument, line: [""]})
+      expect(resultHome._attributes).toEqual(expected)
+    })
 })
 
 describe("convertGender should return correct gender", () => {
-  test("valid fhirGender returns correct hl7 gender", () => {
-    expect(demographics.convertGender("male")).toEqual(codes.SexCode.MALE)
-    expect(demographics.convertGender("female")).toEqual(codes.SexCode.FEMALE)
-    expect(demographics.convertGender("other")).toEqual(codes.SexCode.INDETERMINATE)
-    expect(demographics.convertGender("unknown")).toEqual(codes.SexCode.UNKNOWN)
-  })
+  const cases = [["male", codes.SexCode.MALE],
```

Comment:
Minor point, but this array should be formatted the same was as the others (with first entry on a new line)